pyproject.toml

```diff
@@ -4,7 +4,34 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "microsoft-python-type-stubs"
-dynamic = ["version"]
+version = "0"
+
+[dependency-groups]
+hygiene = ["ruff ==0.11.*"]
+tests = [
+    # Tools used for testing
+    "docopt-ng",
+    "mypy ==1.13.*",
+    "pyright",
+
+    # Typed libraries and stubs
+    "matplotlib >=3.8",
```

Comment:
This isn't a new issue, but I'm curious why we're testing against `matplotlib >=3.8` since those are the versions for which `matplotlib` ships their own stubs. `3.8.0` shipped back in September 2023. I wonder when we'll be comfortable removing our `matplotlib` stubs.